documents/decimal_test.go

```diff
@@ -0,0 +1,139 @@
+// +build unit
+
+package documents
+
+import (
+	"testing"
+
+	"github.com/centrifuge/go-centrifuge/errors"
+	"github.com/centrifuge/go-centrifuge/utils"
+	"github.com/shopspring/decimal"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDecimal(t *testing.T) {
+	tests := []struct {
```

Comment:
we need arithmatic tests here to verify that calculations on decimals and then conversions work